docs/setting-up/server/podman.md

```diff
@@ -0,0 +1,332 @@
+# Podman
```

Comment:
we need to make it clear - this is a "Technical review," possible changes in future